receiver/otlpreceiver/internal/errors/errors.go

```diff
@@ -17,7 +17,7 @@ func GetStatusFromError(err error) error {
 		// https://github.com/open-telemetry/opentelemetry-proto/blob/main/docs/specification.md#failures
 		code := codes.Unavailable
 		if consumererror.IsPermanent(err) {
-			code = codes.InvalidArgument
+			code = codes.Internal
```

Comment:
```suggestion
			// If an error is permanent but doesn't have an attached gRPC status, assume it is server-side.
			code = codes.Internal
```
Or wording of your choice. I just think we should leave a note on why we chose this code since I don't think the spec link above will adequately explain it.